### DISCUSSION TITLE
Dual Audio Synchronized Clips

### DISCUSSION THREAD

**Discussion by ghost** [latenitefilms/BRAWToolbox#179] [Support] [answered]
I'm new to BRAWToolbox. I recorded scratch audio on camera and separate audio on my field recorder. After import into FCP, is there an easy way to swap my audio in the synchronized clips? Or perhaps that should be done during import? What's the best way to do this...use my field recorder audio files with my BRAW files?

**Comment by latenitefilms** [comment]
I would normally get BRAW Toolbox to create Synchronised Clips (simply deselect "Create Multicam Clips" in the Workflow Extension [Settings](https://brawtoolbox.io/workflow-extension/#settings)), then use Sync-N-Link to sync these synchronised clips to my location audio via Timecode - creating new Multicam Clips. Hope this helps!

**Reply by ghost** [reply]
Ahh, I would have to use a timecode generator and sync. OK. I was hoping to just use FCPs synchronized clip function using audio matching to sync them together.

**Reply by latenitefilms** [reply]
You can absolutely still do that manually in Final Cut Pro. Just select your BRAW Toolbox Synchronised Clips and the audio, then create a new Multicam.

**Reply by ghost** [reply]
Ahh okay. Thanks for the quick response!

**Comment by MathewWelsh** [comment]
Hi there, I'm brand new to the BRAW Toolbox, and am a relative newbie on FCPro. I've edited a few projects. I also have a dual system project to ingest and prep for editing. 

When I was a bit apprehensive about going with his blackmagic camera, the camera guy said "just use BRAW toolbox, it's great." The sound guy said "just use timecode to sync my audio files, it'll be great!" If only life were that easy. I'm trying to sync picture to audio in Final Cut Pro. However, when BRAW Toolbox prepares the files to import into FCP, for some reason, all the camera files brought over to FCP appear in the FCP Event as sync'ed clips already -- the camera files have that "linked infinity icon" on the clip. And the audio that is attached to the clip is just the crappy camera mic audio. If I select the camera file and the corresponding audio clip and right-click, the synchronize command is greyed out. That means I cannot go through the usual process to sync the clips to the separate audio files from the field recorder. 

I have just started to watch your (1 hr+) YouTube interview with the Ripple Training guys from last year, but in the summary notes for that video I don't see a topic about working with audio. Also, I've tried looking at what you suggested above, to ahsilver, and I don't see a way to deselect Multicam clips. You recommended Sync-N-Link ($200 USD). I had also looked at Tentacle. But those seem to output an XML and a camera guy did that for me earlier this year on a project, but I seem to remember it just loads one big clip that has to be loaded onto the project timeline. It doesn't maintain separate clips that remain intact single clips that can be viewed in the Browser window.

Are there other settings in BRAW Toolbox to not bring over audio when importing? I cannot find such a thing in any of the settings/buttons. Your help will be much appreciated. Thanks, Mathew
<img width="1198" alt="Screenshot 2024-11-20 at 6 09 02 PM" src="https://github.com/user-attachments/assets/b947bdfb-2c46-461e-9044-68980c7b9c89">

**Reply by latenitefilms** [reply, marked as answer]
You just need to make sure you deselect **Create Multicam Clips** in the Workflow Extension:

<img width="411" alt="image" src="https://github.com/user-attachments/assets/7d43640e-2a2f-4bbd-bd74-555b93aa0178">

...then you can use Sync-N-Link as normal.

See: https://brawtoolbox.io/faq/#does-braw-toolbox-work-with-sync-n-link

**Reply by MathewWelsh** [reply]
Thanks for your reply and the suggestion. I tried this, but this doesn't seem to be the issue. The "Create Multicam Clips" option was not selected in the first place, so I could not deselect it. However, I did make sure that it was off. Then I imported the BRAW clip over to FCP, and the clip still appears as a sync'ed clip. When I double click the video clip, it appears on the timeline as video and audio file, separate. When I right-click on the audio clip, I cannot detach audio. 

It would be so awesome to get a solution for this. I wanted to start editing this project Monday. Other things have held me up, but this is a real stopper. The camera audio on the BRAW clip is getting in the way of pairing/sync'ing field recorder audio to the video picture. I need to remove that audio, or make it a normal part of the video clip. I'm not sure why the import process from BRAW Toolbox to FCP results in video clips that are "sync'ed" clips already. 
<img width="2225" alt="Screenshot 2024-11-22 at 5 26 25 PM" src="https://github.com/user-attachments/assets/bee93789-5cba-4574-a88c-5dd7b45f7092">
<img width="1086" alt="Screenshot 2024-11-22 at 5 34 23 PM" src="https://github.com/user-attachments/assets/9ebd9251-9c61-4d49-9bce-a84bdee85ca3">

**Reply by MathewWelsh** [reply]
Okay, I think I've figured out a way to work this. I import the video files, which come into FCP as sync'ed files. But then I select video files and their corresponding field recorder files (it helps when your sound recordist is on the ball and creates helpful file names for each set up). I highlight the video files and audio files I want to link then I right-click to create Multicam.  That creates a multicam clip. FCP 11 has the handy function where you can hide the files that go into the multicam, so that as you create your mulcticam clips, the source files hide. Then I can see the multicam clip in the browser and I can edit from that. Of course, you have to go into the inspector and turn on/off the audio channels you want to hear. So I think I'm back in business. Cheers, Mathew
